frontend/__mocks__/modules/fluent__react.tsx

```diff
@@ -10,12 +10,19 @@ export const mockFluentReact = {
       },
     };
   },
-  Localized: (props: { children: ReactNode; vars?: Record<string, string> }) =>
+  Localized: (props: {
+    children: ReactNode;
+    id: string;
+    vars?: Record<string, string>;
+  }) =>
     isValidElement(props.children) ? (
       cloneElement(
         props.children,
         {},
-        <>[Localized with vars: {JSON.stringify(props.vars ?? {})}]</>
+        <>
+          [&lt;Localized&gt; with id [{props.id}] and vars:{" "}
+          {JSON.stringify(props.vars ?? {})}]
+        </>
```

Comment:
I updated the mock `<Localized>` element to also display the ID of the string that it outputs, as it was a bit hard to debug without it. Don't know why I didn't add that in the first place. (And also turned the log into `<Localized>` rather than `Localized`, to clarify that it's a component.)